src/librustdoc/clean/auto_trait.rs

```diff
@@ -580,7 +579,9 @@ impl<'a, 'tcx> AutoTraitFinder<'a, 'tcx> {
                                 } => {
                                     let mut new_trait_path = trait_path.clone();
 
-                                    if self.is_fn_ty(tcx, trait_) && left_name == FN_OUTPUT_NAME {
+                                    if self.is_fn_ty(tcx, trait_)
+                                        && left_name == Symbol::intern(FN_OUTPUT_NAME)
```

Comment:
Rather than interning this every time, can you use [`sym::Output`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_span/symbol/sym/constant.Output.html)?
```suggestion
                                        && left_name == sym::Output
```
I think you could also get rid of `FN_OUTPUT_NAME` at the same time.

Reply:
Oh wow. I didn't even know it existed. That's awesome!